practice-two/src/styles/modules/title.scss

```diff
@@ -27,6 +31,10 @@
     &--center {
```

Comment:
What is different between the title-secondary--center and title-secondary--left?